easybuild/tools/options.py

```diff
@@ -550,6 +550,10 @@ def informative_options(self):
 
     def github_options(self):
         """GitHub integration configuration options."""
+        github_list_pr_states = ['open', 'closed', 'all']
+        github_list_pr_orders = ['created', 'updated', 'popularity', 'long-running']
+        github_list_pr_directions = ['asc', 'desc']
```

Comment:
Should we use constants for these (especially since we reuse some of these strings for default settings)?

If so, they should be defined in `tools/github.py` and imported here (since the values are determined by the GitHub API, not by us).